### PR TITLE
updated az pipeline

### PR DESCRIPTION
**Pull request by mizrael**
updated the AZ pipeline to include the new Persistence project.

### CODE REVIEW

File: src/PASopa_pack.sln [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (petrochuk) [inline src/PASopa_pack.sln:2]: Why do we need separate solution file?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -113,6 +113,57 @@ jobs:
         ]
       ServiceEndpointUrl: 'https://api.esrp.microsoft.com/api/v1'
 
+  - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
```
Comment (petrochuk) [inline azure-pipelines.yml:116]: Can we use just one ESRP task?

File: src/PASopa_pack.sln (hunk 1 above)
Comment (mizrael) [reply src/PASopa_pack.sln:2]: this includes only the projects that need to be published as nuget packages, avoids running the pack job for each one

File: azure-pipelines.yml (hunk 2 above)
Comment (mizrael) [reply azure-pipelines.yml:116]: haven't found a decent way to do that (yet)

Comment (mizrael) [reply azure-pipelines.yml]: yes, just updated the code

**Review by tehcrashxor — APPROVED** [review]

File: src/PASopa_pack.sln (hunk 1 above)
Comment (petrochuk) [reply src/PASopa_pack.sln:2]: let's see if we can avoid extra solution

Comment (mizrael) [reply src/PASopa_pack.sln]: added an array parameter to the `pack` step to specify the projects to include

**Pull request merged**